Source/ACE/Entity/DebugObject.cs

```diff
@@ -16,6 +16,67 @@ public DebugObject(ObjectType type, ObjectGuid guid, string name, ushort weenieC
         {
         }
 
+        public DebugObject(ObjectGuid guid, ObjectDescriptionFlag descriptionFlag, BaseAceObject baseAceObject)
+            : base((ObjectType)baseAceObject.TypeId, guid)
+        {
```

Comment:
why duplicate all this code instead of chaining the constructors?  just curious.